### PR TITLE
Review #1

### PR DESCRIPTION
**Pull request by dooya4**

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -16,7 +17,9 @@ document.querySelector(".introduce").onclick = () => {
   slidesDiv.classList.remove("hidden");
   index = 0;
   show(index);
+ 
   console.log("into slides");
+
```
Comment (doyamd) [inline popup.js:22]: 'console.log' not necessary, could be removed to make it more clean

File: popup.js [hunk 2]
```diff
@@ -25,7 +28,9 @@ document.querySelector(".change").onclick = () => {
 };
 
 function show(index) {
+
   console.log(index);
+
```
Comment (doyamd) [inline popup.js:33]: 'console.log' not necessary, could be removed to make it more clean

File: popup.js [hunk 3]
```diff
@@ -44,6 +49,7 @@ nexts.forEach((next) => {
     show(index);
   };
 });
+
 previouses.forEach((previous) => {
   previous.onclick = () => {
     index -= 1;
```
Comment (doyamd) [inline popup.js:55]: the variable name 'previouses' could be substituted to previousButtons. Make it more descriptive.

File: popup.js [hunk 4]
```diff
@@ -52,7 +58,9 @@ previouses.forEach((previous) => {
 });
 
 async function setShortcuts() {
+  
   const extensionShortcuts = await chrome.commands.getAll();
+
```
Comment (doyamd) [inline popup.js:63]: There should be an error handling mechanism to handle if the 'getAll()' command fails.

**Review by dooya4 — COMMENTED** [review]
Overall it is a neat code, with good and descriptive variable naming.
And does the intended work.

**Pull request closed without merging**